packages/airnode-utilities/src/caching/index.ts

```diff
@@ -108,22 +107,33 @@ const addKey = (key: string, data: any, force = false) => {
   }
 
   const filePath = join(CACHE_BASE_PATH, key);
-  try {
-    const fileExists = existsSync(filePath);
-    if (fileExists && !force) {
-      logger.error(`Unable to overwrite key from fs cache as key exists and force is set to false`);
-      logger.error(`Key: ${key}`);
-      return;
-    }
 
-    if (fileExists) {
-      rmSync(filePath, { force: true });
+  const goFileExists = goSync(() => existsSync(filePath));
```

Comment:
Can this actually fail? I think `existsSync` does not throw an error.

Reply:
I think you are right. I'll remove this.